tests/data/shrink_cat.py

```diff
@@ -10,18 +11,21 @@
                     'base_GaussianFlux_magErr', 'base_GaussianFlux_snr', 'e1', 'e2', 'psf_e1', 'psf_e2',
                     'filt', 'detect_isPrimary']
 
+log = logging.getLogger(__name__)
+
 
 def shrinkCat(infile, outfile, verbose=False):
+
     cat = SimpleCatalog.readFits(infile)
     for name in cat.schema.getNames():
         if name in required_columns or 'flag' in name:  # Flags should compress anyway
             continue
         if verbose:
-            print(f"Shrinking {name}")
+            log.debug(f"Shrinking {name}")
```

Comment:
Unfortunately python logger wants f-string vs %-formatting to be a global setting so this would have to be:

```
log.debug("Shrinking %s", name)
```